apps/condo/lang/ru/ru.json

```diff
@@ -1136,6 +1136,19 @@
   "AddItemUsingFileUpload": "Через загрузку файла",
   "Status": "Статус",
   "Account": "Аккаунт",
+  "news.component.RecipientCounter.mailing": "Рассылка",
+  "news.component.RecipientCounter.label.properties": "дома",
+  "news.component.RecipientCounter.willReceive.label": "получат",
+  "news.component.RecipientCounter.willReceive.hint": "В {count} помещениях есть \u2028пользователи мобильного \u2028приложения Doma — они получат \u2028новость",
```

Comment:
Will this translation behave well with any numbers?
How about numbers 0, 1, 21, 31, ...?

Reply:
done